apps/src/templates/instructions/codeReview/CommentEditor.jsx

```diff
@@ -1,20 +1,33 @@
 import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import $ from 'jquery';
+import {getStore} from '@cdo/apps/redux';
 import msg from '@cdo/locale';
 import javalabMsg from '@cdo/javalab/locale';
 import Button from '@cdo/apps/templates/Button';
 import color from '@cdo/apps/util/color';
 
 export default class CommentEditor extends Component {
-  state = {
-    comment: '',
-    isEditing: false
+  static propTypes = {
+    onNewCommentSubmit: PropTypes.func.isRequired,
+    token: PropTypes.string.isRequired
   };
 
-  commentChanged = event => {
-    this.setState({
-      comment: event.target.value,
-      isEditing: true
-    });
+  state = {comment: ''};
```

Comment:
Removing `isEditing` is an unrelated change that just makes it so when the comment is blank, the buttons disappear (instead of having a dedicated `isEditing` flag)